planner/core/explain.go

```diff
@@ -431,7 +431,7 @@ func (p *PhysicalIndexReader) ExplainInfo() string {
 
 // ExplainNormalizedInfo implements Plan interface.
 func (p *PhysicalIndexReader) ExplainNormalizedInfo() string {
```

Comment:
Can you provide tests to verify and ensure that people in future won't change it by mistake?

Reply:
I have already add a failpoint-test here, I already tested without this change, some plan normalized test will be failed.